src/main/java/reactor/kafka/receiver/ReceiverPartition.java

```diff
@@ -48,6 +48,13 @@
      */
     void seek(long offset);
 
+    /**
+     * Seek to the topic partition offset that is greater than or equal to the timestamp.
```

Comment:
should it clearly specify that if no such offset is found, this is equivalent to `seekToEnd()`?